app/controllers/conference_attendances_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ConferenceAttendancesController < ApplicationController
+
+  def update
+    @conference_attendance = ConferenceAttendance.find(attendance_params[:id])
+    @team = Team.find(attendance_params[:team_id])
```

Comment:
I think it would make more sense to not set @team separately dependent on the params but to use `@conference_attendance.team` for the redirects later on. This would communicate more clearly that the team that we are redirecting to is the team that is associated to this `@ conference_attendance` object.